lighthouse-core/lib/url-shim.js

```diff
@@ -23,7 +23,7 @@
 
 /* global self */
 
-const ELLIPSIS = '\u2026';
+const Util = require('../report/v2/renderer/util.js');
```

Comment:
lol, you and i came up with this solution together. we can do a new file. it's just more...files.